packages/lesswrong/components/posts/PostsPreviewTooltip.tsx

```diff
@@ -82,7 +85,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     marginTop: theme.spacing.unit,
     marginLeft: -13,
     marginRight: -13,
-    marginBottom: -9
+    marginBottom: -commentsNodeRootMarginBottom
```

Comment:
This negative margin was set wrong, probably because someone edited the comment margins that it's designed to reverse. I updated the comment styling to use a variable so it won't get out of sync again. Probably at some point the other negative-margins in this thing should also get updated.